content/blog/2019/11/2019-11-21-jenkins-health-advisor-by-cloudbees.adoc

```diff
@@ -0,0 +1,50 @@
+
+That’s why we created Jenkins Health Advisor by CloudBees. 
+Here at https://www.cloudbees.com?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[CloudBees], we have years of experience https://www.cloudbees.com/products/cloudbees-jenkins-support?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[supporting our customers who are using Jenkins], including our proprietary products build on top of Jenkins like https://www.cloudbees.com/products/cloudbees-core?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[CloudBees Core]. 
+As a result, our support team is made up of automation experts with Jenkins knowledge you can’t get anywhere else. 
+
+Automated health checks started when our support engineers created a platform so they could write rules to detect known issues on https://plugins.jenkins.io/support-core[support bundles] provided by our customers, and redirect them to the required knowledge source to diagnose and resolve the issue. 
+
+After years of internal usage we decided to share this service with the community and we are pleased to introduce: https://www.cloudbees.com/jenkins-health-advisor?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[Jenkins Health Advisor by CloudBees].
+
+[.text-center]
+image:/images/post-images/jenkins-health-advisor-by-cloudbees/logo.svg[Jenkins Health Advisor by CloudBees logo,width=256,link="https://plugins.jenkins.io/cloudbees-jenkins-advisor"]
+
+https://www.cloudbees.com/jenkins-health-advisor?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[Jenkins Health Advisor by CloudBees] can detect a large range of issues from simple configuration issues to security and best practices concerns - all critical elements of Jenkins implementations. 
+Getting started is done in https://www.cloudbees.com/jenkins-health-advisor?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin#download[3 steps], and within 24 hours you will receive your first report.
+
+https://www.cloudbees.com/jenkins-health-advisor?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[Jenkins Health Advisor by CloudBees] automatically analyzes your Jenkins environment, proactively identifies potential issues and advises you of solutions with detailed email reports:
+
+[.text-center]
+image:/images/post-images/jenkins-health-advisor-by-cloudbees/report.png[Jenkins Health Advisor by CloudBees report,width=512]
+
+We hope that you will appreciate this service and it will help you to keep your masters healthy. 
+Take a few minutes to read our https://docs.cloudbees.com/docs/admin-resources/latest/plugins/cloudbees-jenkins-advisor?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[documentation], discover the service and don’t hesitate to contact us on the jenkins community channels.
```

Comment:
```suggestion
Take a few minutes to read our https://docs.cloudbees.com/docs/admin-resources/latest/plugins/cloudbees-jenkins-advisor?utm_medium=blog&utm_source=jenkins.io&utm_campaign=cloudbees-jenkins-advisor-plugin[documentation], discover the service and don’t hesitate to contact us on the Jenkins community channels.
```

link channels BTW?

Reply:
What do you advise to use @oleg-nenashev ?  I am and can be anywhere....

Reply:
Maybe a Gitter chat.?
I have also heard some rumors about a CloudBees community portal: https://support.cloudbees.com/hc/en-us/community/topics

Reply:
Ok, I will mention gitter + emails